source/gsoc.md

```diff
@@ -26,10 +26,11 @@ Please watch this page for updates concerning our participating in 2021.
 * Watch [GSoC video presentation](https://www.youtube.com/watch?v=S6IP_6HG2QE)
  for students.
 * Examine the list of [available ASF GSoC 2021 projects](https://s.apache.org/gsoc2021ideas)
-	* Note that you can search the issue tracker for GSoC projects by
+	* <span class="bg-info">NOTE that you can search the issue tracker for GSoC projects by
 filtering using the "gsoc2021" label, this allows you to narrow down the list
-to the projects you are interested in.
+to the projects you are interested in. https://issues.apache.org/jira/issues/?jql=labels+%3D+gsoc2021</span>
```

Comment:
```suggestion
to the projects you are interested in. 
Or simply use this link https://issues.apache.org/jira/issues/?jql=labels+%3D+gsoc2021 .</span>
```
Keeping this in separate line may improve rendering and readability. WDYT?

Reply:
Should look better now :)
Thanks!